chore: update note for time chaos (#177)
chaos-mesh/website: docs/simulate-time-chaos-on-kubernetes.md

Chaos Mesh provides the TimeChaos experiment type. You can use this experiment type to simulate a time offset scenario. This document describes how to create a TimeChaos experiment and its associated configuration file.

> ::: note
> 
> TimeChaos only affects the PID `1` process in the container's PID namespace, and child processes of the PID `1`. For example, one process started by `kubectl exec` would not be affected.

Reviewer: ```suggestion
::: note

TimeChaos only affects the PID `1` process in the PID namespace of the container, and child processes of the PID `1`. For example, the process started by `kubectl exec` does not be affected.
```
